assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/osgi/log4j2.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+   <Appenders>
+      <RollingFile name="PENTAHOFILE" fileName="../logs/osgi_pentaho.log"
+                   filePattern="../logs/osgi_pentaho.%d{yyyy-MM-dd}.log">
+         <PatternLayout>
+            <Pattern>%d %-5p [%c] %m%n</Pattern>
+         </PatternLayout>
+         <Policies>
+            <TimeBasedTriggeringPolicy />
+         </Policies>
+         <DefaultRolloverStrategy />
+      </RollingFile>
+      <Console name="PENTAHOCONSOLE">
```

Comment:
This is missing the ThresholdFilter  set to ERROR.  The old log4j.xml had this in there.